server/sonar-server/src/main/java/org/sonar/server/computation/issue/ScmAccountToUserLoader.java

```diff
@@ -36,17 +36,17 @@
 /**
  * Loads the association between a SCM account and a SQ user
  */
-public class ScmAccountCacheLoader implements CacheLoader<String, String> {
+public class ScmAccountToUserLoader implements CacheLoader<String, String> {
 
   private final Logger log;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make the "log" logger private static final and rename it to comply with the format "LOG(?:GER)?". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1312)